developer-extension/src/panel/app.tsx

```diff
@@ -1,18 +1,11 @@
-import { Provider as BumbagProvider, css, Box } from 'bumbag'
+import { Provider as BumbagProvider, Box } from 'bumbag'
 import React, { Suspense } from 'react'
 
 import { Panel } from './panel'
 
 const theme = {
   global: {
     fontSize: 14,
-    styles: {
-      base: css`
-        body {
-          width: 300px;
-        }
-      `,
-    },
```

Comment:
Could you keep a global style like that:
```ts
    styles: {
      base: css`
        body {
          min-height: 100vh;
        }
      `,
```
so the `body` background color is applied to the whole panel? This is mostly visible for dark themes:
![Screenshot 2021-11-22 at 14 43 07](https://user-images.githubusercontent.com/61787/142871972-f828aa90-cfe5-4cf0-888a-4e2145e65be2.png)

Reply:
This issue is actually fixed by newer version of bumbag but newer version of bumbag has type issues 😢 
I used the css trick.

Reply:
Ok, we'll upgrade bumbag someday.